client/my-sites/checkout/composite-checkout/components/wp-order-review-line-items.js

```diff
@@ -538,13 +538,13 @@ function LineItemSublabelAndPrice( { item } ) {
 		isGSuiteOrExtraLicenseProductSlug( productSlug ) || isGoogleWorkspaceProductSlug( productSlug );
 
 	if ( item.type === 'plan' && item.wpcom_meta?.months_per_bill_period > 1 ) {
-		return translate( '%(sublabel)s: %(monthlyPrice)s /month × %(monthsPerBillPeriod)s', {
+		return translate( '%(sublabel)s: %(monthlyPrice)s/month × %(monthsPerBillPeriod)s', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.